src/main/java/ru/smirnovv/FindMax.java

```diff
@@ -4,21 +4,31 @@
 import picocli.CommandLine;
 
 import java.io.BufferedReader;
+import java.io.IOException;
 import java.io.InputStreamReader;
 
 import static java.lang.System.err;
 import static java.lang.System.out;
 import static picocli.CommandLine.*;
 import static ru.smirnovv.MyCollectors.maxN;
 
+/**
+ * Program for find the given number of maximums.
+ */
 @Command(description = "Finds the given number of maximums.",
         name = "FindMax", mixinStandardHelpOptions = true,
         version = "FindMax 0.1.0")
```

Comment:
Would be great to replace the name and version attributes with "${project.artifactId}" and "${project.version}" accordingly and to enable filtering of java source files in Maven that will place the actual artifactId and version into the attribute values. Another way is just to add the version info into the Manifest file and to implement IVersionProvider from Picocli that should extract the version info from the Manifest file in runtime.